bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "adapt-contrib-matching",
-  "version": "2.1.1",
+  "version": "2.1.2",
```

Comment:
Yeah, thanks, I just noticed that when I created the PR -- there are quite a few waiting to go in.  It should still merge OK though.